docs/install/centos/services/mariadb.md

```diff
@@ -18,22 +18,16 @@ Django also requires development libraries from MariaDB
 
 
 ## Schema and Access Control
-
-Create schema `openoni`
-
-Add `openoni` user only connecting from `localhost`
-
-Grant `openoni` user all privileges except `GRANT OPTION`
-on `openoni%` schema(s)
-
 `mysql -u root -p`:
 
 ```bash
+-- Create the database and switch to use it
 CREATE SCHEMA `openoni`;
-
 USE openoni;
 
-CREATE USER openoni@localhost IDENTIFIED BY 'password';
+-- Add a user which can only connect locally
+CREATE USER 'openoni'@localhost IDENTIFIED BY '(passphrase)';
```

Comment:
this looks a little confusing to me, like that I should type parenthesis around my password / passphrase

Reply:
Agreed - everywhere I see examples, it's `IDENTIFIED BY 'password'`, and sometimes a specific password with docs stating something like "replace XYZ with your actual password".  I don't think the parens are a good idea.

Reply:
Ah okay. Duly noted. I can explain in the comment and omit the parens. Thanks :)